app/assets/stylesheets/components.scss

```diff
@@ -115,15 +115,52 @@
 }
 
 .circuit-card-name {
-  display: inline-block;
   float: left;
   font-weight: 600;
-  margin-top: 10px;
-  max-height: 30px;
-  max-width: 150px;
-  overflow: hidden;
+  margin-top: 5px;
+  position: relative;
   text-align: left;
-  text-overflow: ellipsis;
```

Comment:
Keep this (or rather move it to inside p tag. (You migth require display: inline-block for this to work)

Reply:
white-space:nowrap made it work, made changes accordingly!!